packages/modelviewer.dev/src/docs-and-examples/create-html.ts

```diff
@@ -136,6 +141,10 @@ function getLowerCaseKey(key: string):
     string {
       if (key === 'CSS Custom Properties') {
         return 'cssProperties';
+      } else if (key === 'Static Methods') {
```

Comment:
Consider a switch statement for this kind of thing.

Reply:
done